CoroutineTemplate/app/src/staging/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,22 @@
+<network-security-config>
+    <debug-overrides>
+        <trust-anchors>
+            <certificates src="user" />
+            <certificates src="system" />
+        </trust-anchors>
+    </debug-overrides>
+
+    <base-config cleartextTrafficPermitted="true">
```

Comment:
I think we don't need this tho? 🤔  or do we need to grant the permission to transmit via `http` somewhere?

Reply:
Aah, I see!

We can remove this indeed, if we don't want to support `http` 🙏